cmd/kubeless/function/deploy.go

```diff
@@ -209,6 +214,7 @@ func init() {
 	deployCmd.Flags().StringP("trigger-topic", "", "", "Deploy a pubsub function to Kubeless")
 	deployCmd.Flags().StringP("schedule", "", "", "Specify schedule in cron format for scheduled function")
 	deployCmd.Flags().StringP("memory", "", "", "Request amount of memory, which is measured in bytes, for the function. It is expressed as a plain integer or a fixed-point interger with one of these suffies: E, P, T, G, M, K, Ei, Pi, Ti, Gi, Mi, Ki")
+	deployCmd.Flags().StringP("cpu", "", "", "Request amount of cpu for the function.")
```

Comment:
you might want to add a sample or some more words to explain the accepted format for the cpu number. https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#meaning-of-cpu